sdk/src/instruction_processor_utils.rs

```diff
@@ -27,6 +27,29 @@ macro_rules! solana_entrypoint(
     )
 );
 
+#[macro_export]
+macro_rules! solana_program(
+    ($id:ident, $bs58:expr, $name:ident, $entrypoint:ident) => (
```

Comment:
Need to document these parameters in detail. Even after looking at the implementation, I can quite figure out the purpose of each. And hopefully adding docs will expose some better parameter names.